mobile.css

```diff
@@ -0,0 +1,109 @@
+.body {
+    max-width: 375px;
+    margin: 0;
+}
+.nav__link {
+    display: none;
+}
+
+.logo {
+    width: 138px;
+    height: 42px;
+}
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 375px;
+    margin-bottom: 30px;
+    margin-top: 5px;
+}
+
+.burger {
+   width: 24px;
+   height: 15px;
+}
+.title {
+    max-width: 206px;
+}
+
+.title__h1 {
+    margin: 0;
+    font-size: 50px;
+}
+
+.title__h2 {
+    font-size: 10px;
+    max-width: 161px;
+    margin: 0 auto;
+}
+.background {
+    max-width: 448px;
+}
+.image__car {
+    max-width: 357px;
+}
+
+.box {
+    margin: 0 auto 34px;
+    max-width: 303px;
+    flex-wrap: wrap;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)